### ISSUE TITLE
Design: Onboarding Ideal Flow

### ISSUE THREAD

**Issue by nuriapenya** [Automattic/wp-calypso#96055]
# Details

This issue is the design definition of the aspired onboarding flow referred to in [Onboarding North Star](p58i-irT-p2). This **Epic** compiles all the steps that need to be done to achieve this iteration.

## The ideal flow

This iteration is focused on removing all unnecessary friction and getting users into the product as quickly as possible. The biggest part is removing domains/plans from the initial stepper to later in the admin. This will be combined with configuring the entire downstream site creation experience, which will be reflected in the home Launchpad.

`Goals` → `Design Choice` (Design Picker or Big Sky) → `Account` → `Home Launchpad` → `Domains` → `Plans`

![Image](https://github.com/user-attachments/assets/3190d3a3-279e-41ab-a4e0-d9fc47ed0b50)

Link to Figma (WIP): W9xI27S6Swvw5Ku21EbZvn-fi-9303_20306

**Comment by nuriapenya** [comment]
### Decision to solve for V1

When moving the Goals screen at the beginning of the flow, what happens with the **Skip to admin** step? Allowing users to skip directly to My Home (without seeing domains/plans) could impact cash flow.

![Image](https://github.com/user-attachments/assets/731c3709-e877-48f9-b162-51ae597f1cad)


**Option 1:**
Goals Screen → {skip to admin} → Account/Login → Domains → Plans → My Home

**Option 2:**
Goals Screen → {skip to admin} → Account/Login → ~~Domains → Plans →~~  My Home

I believe the idea in the long term is moving domains/plans later in the flow, so this could be a first step towards that, but I am looking for validation.

**Comment by fditrapani** [comment]
> When moving the Goals screen at the beginning of the flow, what happens with the Skip to admin step? We agreed we will direct users to My Home, but in doing so, should we skip Domains/Plans?

I think it would be worthwhile to test option 2 for some time as long as we have domain + plan steps in the launchpad that replace those options or we shows the domain + plans steps when you launch the site.

**Comment by isatuncman-auto** [comment]
> as long as we have domain + plan steps in the launchpad that replace those options or we shows the domain + plans steps when you launch the site.

I completely agree with this. We must ensure a clear and seamless free-to-paid conversion process and domain presentation experience post-signup before enabling users to skip the domain or plan pages. Additionally, we should strategically define the domain types users can choose from before launching a free or paid site.

**Comment by annezazu** [comment]
>We must ensure a clear and seamless free-to-paid conversion process and domain presentation experience post-signup before enabling users to skip the domain or plan pages.

Agreed with this too. We just aren't there yet in terms of dropping folks into a more seamless upgrade process otherwise.

**Comment by nuriapenya** [comment]
@fditrapani @annezazu @isatuncman-auto 👋🏼  I added here some starting ideas to ensure we are surfacing these two things right after the initial sign-up: [Onboarding: Launchpad & My Home – i1](https://wp.me/p9Jlb4-fb3)

Some key ideas:

* Having dedicated areas on My Home where Domains or Launch flows are more outstanding
* Ensure we drive users to the correct places where more contextual upsell nudges are already in place (e.g. Editor, Preview, etc.)
* Create goal-specific upsell nudges (e.g. Video -> Storage, Newsletter -> Stats) 

On parallel, I also opened this issue to ensure we are looking at the upsell topic more holistically:

https://github.com/Automattic/wp-calypso/issues/96875

**Comment by nuriapenya** [comment]
### Update for the MVP (20 Dec)

* `Skip to admin` option will still go through Domains/Plans as long as we don't have a more defined upsells solution built in the Launchpad/My home. 

Goals Screen → {skip to admin} → Account/Login → Domains → Plans → My Home
